modules/ROOT/pages/footnotes.adoc

```diff
@@ -0,0 +1,50 @@
+= Footnotes plugin
+:navtitle: Footnotes
+:description: Insert footnotes into the {productname} editor
+:keywords: footnotes, footnote, insertfootnotes, insert, footnotesupdate, mceInsertFootnote, mceUpdateFootnotes
+:pluginname: Footnotes
+:plugincode: footnotes
+:pluginminimumplan: tierthree
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a footnote into the {productname} editor.
+
+Invoking the plugin adds a superscripted footnote number at the current insertion point, or at the end of the current selection.
+
+The plugin then places the insertion point in a new line in the footnotes section located at the bottom of the editor, allowing the user to add the footnote entry related to the newly-created footnote.
+
+== Interactive Example
+
+liveDemo::footnotes[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'footnotes',
+  toolbar: 'footnotes',
+});
+----
+
+A footnotes section is automatically created and updated whenever a footnote is inserted into the {productname} editor.
```

Comment:
Is the footnotes section also updated when a footnote is removed?

Reply:
```suggestion
A footnotes section is automatically created or updated whenever a footnote is inserted, reordered, or removed within the {productname} editor.
```
How does this sound?